src/rules/tsrDetectPossibleTimingAttacksRule.ts

```diff
@@ -1,13 +1,79 @@
 import * as ts from 'typescript';
 import * as Lint from 'tslint';
 
-const keywordMask = new RegExp(
-    '^((' + ['password', 'secret', 'api', 'apiKey', 'token', 'auth', 'pass', 'hash'].join(')|(') + '))$',
-    'im'
-);
+const keywordMask = new RegExp('^.*((' + [
+    'password',
+    'secret',
+    'api',
+    'apiKey',
```

Comment:
By accident, I've merged your change on this line, but I think we should add `auth` keyword back into this list

Reply:
I'm fine with that. Added back